src/views/EventPage.vue

```diff
@@ -0,0 +1,266 @@
+<template>
+  <div class="page-container">
+    <!-- background img -->
+    <div class="background-img">
+      <div class="gradient-overlay" />
+      <img :src="event.mediaLink.cover" alt class="background-img" />
+    </div>
+
+    <NavBtns v-bind="navOptions" class="nav-btns" />
+    <!-- <ExploreBar /> -->
+
+    <!-- event details -->
+    <div v-rellax="{ speed: 10 }" class="event-container">
+      <!-- gallery -->
+      <div class="gallery">
+        <div
+          v-for="(img, index) in activeImages"
+          :key="`tag-${index}`"
+          class="image-container"
+        >
+          <img :src="img" alt />
+        </div>
+        <div v-if="event.mediaLink.host.length > 3" class="image-container">
+          <h4>+{{ event.mediaLink.host.length - 3 }}</h4>
```

Comment:
Will want to start an issue for a 'gallery/picture viewer' when you click on these